jni/audioengine.cpp

```diff
@@ -256,9 +256,10 @@ namespace AudioEngine
             bool mustCache        = AudioEngineProps::CHANNEL_CACHING && channel->canCache() && !isCached; // whether to cache this channels output
             int cacheReadPos      = 0;  // the offset we start ready from the channel buffer (when writing to cache)
 
-            SAMPLE_TYPE channelVolume                = ( SAMPLE_TYPE ) channel->getVolumeLogarithmic();
             std::vector<BaseAudioEvent*> audioEvents = channel->audioEvents;
-            int amount                               = audioEvents.size();
+            int amount = audioEvents.size();
+
+            SAMPLE_TYPE channelVolume = ( SAMPLE_TYPE ) channel->getVolumeLogarithmic() / ( SAMPLE_TYPE ) channelAmount;
 
```

Comment:
Each AudioChannels volume is divided by the total amount of channels to provide headroom